themes/ee/asset/javascript/src/common.js

```diff
@@ -9,6 +9,14 @@
 
 $(document).ready(function(){
 
+	$('.js-dropdown-toggle, .select__dropdown-item').each(function() {
+		$(this).keyup(function(event) {
+			if (event.keyCode === 13) {
```

Comment:
Could you leave a comment labeling keycode 13? It's hard to read in passing without it.

Reply:
Left a comment in the file. keyCode 13 => Enter

Reply:
Thanks!